.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Debug Tests",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "purpose": [
+                "debug-test"
+            ],
+            "console": "integratedTerminal",
+            "justMyCode": false,
+            "env": {
+                "PYTEST_ADDOPTS": "-xvs -n 1"
```

Comment:
mainly to make debugging via test explorer in vscode more convenient - otherwise it will launch tests with default xdist -n auto. `justMyCode` also allows stepping into internal methods - often useful for debugging